### PR TITLE
DOC:updated pie plot legend overlap issue, closes #34288

### LINKED ISSUE #34288: DOC: improve pie chart example
#### Location of the documentation

https://pandas.pydata.org/pandas-docs/dev/reference/api/pandas.DataFrame.plot.pie.html

**Note**: You can check the latest versions of the docs on `master` [here](https://pandas.pydata.org/docs/dev/).

#### Documentation problem

This example doesn't look great:
image

The legend covers the labels, and the two pies are so close that "mercury" is hard to read.

#### Suggested fix for documentation

Make a more readable plot

### PR DESCRIPTION
**Pull request by wicky1234444**
- [X] closes #34288
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/plotting/_core.py [hunk 1]
```diff
@@ -1475,7 +1475,7 @@ def pie(self, **kwargs):
         .. plot::
             :context: close-figs
 
-            >>> plot = df.plot.pie(subplots=True, figsize=(6, 3))
+            >>> plot = df.plot.pie(subplots=True, figsize=(11, 11))
```
Comment (MarcoGorelli) [inline pandas/plotting/_core.py:1478]: Like this there's lots of whitespace above and below, perhaps do this?

```suggestion
            >>> plot = df.plot.pie(subplots=True, figsize=(11, 5))
```

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Thanks @wicky1234444 , this looks better

Did you manage to build this page of the documentation locally?

Also, please put the issue number in the description (`closes #<issue number goes here>`)

File: pandas/plotting/_core.py (hunk 1 above)
Comment (wicky1234444) [reply pandas/plotting/_core.py:1478]: yeah. I dropping a link to the image here.
pandas-Series-plot-pie-2

Comment (wicky1234444) [reply pandas/plotting/_core.py:1478]: ok, i will change that.

Comment (wicky1234444) [reply pandas/plotting/_core.py]: sorry i forgot add this in the description . closes #34288

Comment (wicky1234444) [reply pandas/plotting/_core.py]: i have added the suggested changes. closes #34288

Comment (MarcoGorelli) [reply pandas/plotting/_core.py]: Great, thanks!

> closes #34288

Could you add this to the pull request description? I don't know if it works as a keyword if it's just in a comment

**Review by WillAyd — APPROVED** [review]
lgtm

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @wicky1234444

**Pull request merged**